test/generators/suspenders/ci_generator_test.rb

```diff
@@ -21,7 +21,7 @@ class CiGeneratorTest < Rails::Generators::TestCase
 
       test "raises if PostgreSQL is not the adapter" do
```

Comment:
1) Should this test description reflect "PostgreSQL or SQLite"?

2) Should this or another test assert that `with_database "sqlite"` does not raise?